README.md

```diff
@@ -12,3 +12,34 @@
 - [How To Build](https://github.com/status-im/status-go/wiki/Build-Process-Explained)
 - [Notes on Bindings](https://github.com/status-im/status-go/wiki/Notes-on-Bindings)
 - [Developers' Guide](https://github.com/status-im/status-react/wiki/Developers'-Guide)
+
+# Intro
+status-go is a underlying part of status mobile app and handles blockchain and Status accounts related code.
+
+It's written in Go and requires Go 1.8 or above.
+
+It uses Makefile to do most common actions. See `make help` output for available commands.
+
+Integration tests and server require `statusd-data` dir to store synched blockchain and other stuff, so we use `build/env.sh` script before running most of the commands in Makefile (like `go test` or running the server)
+
+# Build
+There are two main modes status-go can be built:
+ - standalone server
+ - library to link for Android or iOS
+
+Use following Makefile commands:
+
+`make statusgo` (builds binary into `build/bin/statusd`)
```

Comment:
Needs a dashed list here for correct rendering.

Reply:
Ouch, thanks!